### ISSUE TITLE
Safari support

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor4-benderjs-runner#15]
It would be nice to have Safari support. When we worked in December on the runner it worked with Travis CI osx image just fine (apart from #10) - see for example ckeditor/ckeditor4-benderjs-runner-tests@fb69346 and [https://travis-ci.org/github/ckeditor/ckeditor4-benderjs-runner-tests/builds/750370942.](https://travis-ci.org/github/ckeditor/ckeditor4-benderjs-runner-tests/builds/750370942.)

After getting back to this in May, Travis was not able to trigger any job containing osx image config. The job simply didn't run and when triggering manual Travis CI was giving were vague error (see related commits in ckeditor/ckeditor4-benderjs-runner-tests#3).

We can try with osx image once again and dig deeper or go with BS straightaway (#4) which should also solve this issue.

**Comment by f1ames** [comment]
Fixed in ckeditor/ckeditor4#4771.

**Comment by f1ames** [comment]
It seems there is still something not working, see https://travis-ci.com/github/ckeditor/ckeditor4/jobs/518650489:

![image](https://user-images.githubusercontent.com/1061942/123275720-dd0af380-d504-11eb-8fd6-f09427e8030b.png)

And even bigger issue is that pretty old OSX and Safari versions are used:

> ProductVersion: 10.13.6
> name: 'safari', version: '11.1.2'

Not sure if Travis is a way to go or we should strongly consider migrating to GHA.

**Comment by f1ames** [comment]
And PR dedicated for testing - ckeditor/ckeditor4#4774.

**Comment by f1ames** [comment]
As mentioned in [https://github.com/ckeditor/ckeditor4-benderjs-runner/issues/15#issuecomment-867659623,](https://github.com/ckeditor/ckeditor4-benderjs-runner/issues/15#issuecomment-867659623,) it's really problematic on Travis so this issue is blocked by GHA migration for now.

**Comment by f1ames** [comment]
Ok, there were some issue at the beginning (as mentioned in https://github.com/ckeditor/ckeditor4-benderjs-runner/issues/15#issuecomment-867659623) but it started to work quite stable now so I'm closing this issue.
